user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Database_security/Security_Opensearch.md

```diff
@@ -10,3 +11,45 @@ uid: Security_OpenSearch
 ## TLS Encryption
 
 For information on how to enable both client-server and inter-node TLS encryption, refer to [TLS configuration](xref:Installing_OpenSearch_database#tls-configuration)
+
+## Upgrading OpenSearch
+
+DataMiner supports both the 1.X and 2.X version range of OpenSearch; however, for new installations, we recommend installing OpenSearch 2.X.
+
+We also recommend that you keep your OpenSearch installation up to date with the latest version within the range you have chosen.
+
+### Minor upgrade
+
+There are two ways to perform a minor upgrade of an OpenSearch cluster:
+
+- [A cluster restart upgrade](#cluster-restart-upgrade)
+- [A rolling upgrade](#rolling-upgrade)
+
+Regardless of which upgrade strategy you choose, you will first need to perform these steps:
+
+1. Download the latest version within your release track: [1.X](https://opensearch.org/lines/1x.html) or [2.X](https://opensearch.org/lines/2x.html).
+1. [Back up the configuration files](https://opensearch.org/docs/latest/install-and-configure/upgrade-opensearch/index/#backing-up-configuration-files).
+1. Back up the cluster by [taking a snapshot](https://opensearch.org/docs/latest/tuning-your-cluster/availability-and-recovery/snapshots/snapshot-restore/).
```

Comment:
We have a procedure for taking snapshots within our documentation: https://docs.dataminer.services/user-guide/Advanced_Functionality/Databases/Indexing_database/OpenSearch_Database/Configuring_OpenSearch_Backups.html#taking-the-snapshot
Could we link to that instead?

Reply:
I was not aware we already had this documented, I'll make the necessary changes